groclient/client.py

```diff
@@ -347,7 +347,7 @@ def get_data_points_generator(self, **selection):
                 if len(missing_params) == 1
                 else ', '.join(missing_params[:-1]) + ' and ' + missing_params[-1] + ' are'
             )
-            self._logger.error(message)
```

Comment:
I'm a bit confused as to why we need logging sent to Kibana from the `api-client` at all, but LGTM.